_docs/tasks/traffic-management-v1alpha3/ingress.md

```diff
@@ -13,7 +13,7 @@ allows users to specify services that should be exposed outside the cluster.
 For traffic entering an Istio service mesh, however, an Istio-aware [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-controllers)
 is needed to allow Istio features, for example, monitoring and route rules, to be applied to traffic entering the cluster.
```

Comment:
Please remove this line as we clearly state that the ingress controller has minimal functionality